src/components/CreditCardInformation.jsx

```diff
@@ -2,8 +2,8 @@ import React, {useState} from 'react';
 import {loadStripe} from "@stripe/stripe-js";
 import {CardElement, Elements, useElements, useStripe} from "@stripe/react-stripe-js";
 import {AlertDialogHeader, Box, Button, Center} from "@chakra-ui/react";
-import useHandlePayment from "../../contexts/useHandlePayment";
-import AlertDialogBox from "../AlertDialogBox";
+import useHandlePayment from "../hooks/useHandlePayment";
+import AlertDialogBox from "./AlertDialogBox";
```

Comment:
I think the stripe id below can go inside an env file

Reply:
refer to comment above